src/index.html

```diff
@@ -13,5 +13,142 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <h2>Personal information</h2>
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="wrapper"> <legend>Personal information</legend>
+        <label class="personal">
+          Surname: 
+          <input 
+            type="text" 
+            name="surname"  
+            autocomplete="off"
+          >  
+        </label>
+        <label class="personal">
+          Name:
+          <input 
+            type="text" 
+            name="name" 
+            autocomplete="off"
+          >  
+        </label>
+        <label class="personal">
+          How old are you?
+          <input
+            type="number" 
+            name="old" 
+            max="100" 
+            min="1" 
+            value="12"
+          >  
+        </label>
+        <label class="personal">
+          Full date of birth:
+          <input
+            type="date" 
+            name="birth"
+            >
+        </label>
+        <label class="personal">
+          I accept the term of the agreement
+          <input 
+            type="checkbox" 
+            name="accept" 
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="wrapper"> <legend>Registration</legend>
+          
+        <label class="registration">
+          E-mail:
+          <input 
+            type="email" 
+            name="email" 
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label class="registration">
+          Password:
+          <input 
+            type="password" 
+            name="password" 
+            minlength="8" 
+            maxlength="32" 
+          >
+        </label>
+      </fieldset>
+        
+      <fieldset class="wrapper"> <legend> An intresting facs about you</legend>
+          
+        <label class="facts">
+          Do you love cats? 
+          <input 
+            type="radio" 
+            name="cats"
+          > Yes
+          <input 
+            type="radio" 
+            name="cats"
+          > No
```

Comment:
Add labels for "yes" and "no" inputs, so they work correctly
![labels](https://user-images.githubusercontent.com/96409563/228154821-cbf72342-c450-4a88-8b4a-de84fc709aca.gif)